doc/sphinx/addendum/miscellaneous-extensions.rst

```diff
@@ -7,20 +7,24 @@ of program refinements. To use the Derive extension it must first be
 required with ``Require Coq.derive.Derive``. When the extension is loaded,
 it provides the following command:
 
-.. cmd:: Derive @ident__1 SuchThat @one_term As @ident__2
-
-   :n:`@ident__1` can appear in :n:`@one_term`. This command opens a new proof
-   presenting the user with a goal for :n:`@one_term` in which the name :n:`@ident__1` is
-   bound to an existential variable :g:`?x` (formally, there are other goals
+.. cmd:: Derive @open_binders in @type as @ident
+         Derive @open_binders SuchThat @type As @ident
+
+   where :n:`@open_binders` declares a list of declarations of the form
+   :g:`@ident__i : @type__i` which can appear in :n:`@type`. This
+   command opens a new proof presenting the user with a goal for
+   :n:`@type` in which each name :n:`@ident__i` is bound to an
+   existential variable :g:`?x__i` (formally, there are other goals
```

Comment:
```suggestion
   :n:`@type` in which each :n:`@ident__i` is bound to an
   existential variable :n:`?x__i` (formally, there are other goals
```

Ah, these will be fresh existential variables, right, so it's misleading to specifically say :n:`?x__i`; they may be numbered differently./

Reply:
Not exactly, the existential variables will have the same name as the constant. Maybe should I used `?name_i` instead?

Reply:
I don't know.  I really don't understand the command.  Sounds like `?ident__i` is sort of right but the numbers may not match up.  Maybe just be less specific, e.g. just say "bound to a new existential variable beginning with :n:`@ident__i` (formally ..." and don't give any `?XXX` in the text.